_configs/eval_ssm.yaml

```diff
@@ -44,10 +44,12 @@ infmax_models:
 
 run:
   experiment_type: "evaluate"
-  nb_seeds: 1
-  repetitions: 30
-  random_seed: 43
-  average_results: False
+  nb_diffusion_seeds: 1
+  nb_repetitions:
+    diffusion: 30
+    stochastic_infmax: 2
+  rng_seed: 43
```

Comment:
[1]
`random_seed` or `random_state` is more "canonical"

Reply:
I agree, but we should pay attention to avoid potential misunderstanding of terms. We have here two seeds (seed set and seed of the randomnumbergenerator). Moreover, I'd like to put into evaluation random seed selection method. Hence I wanted to make it clearly distinguishable